app/assets/main/react_components/routes/lifestyle_footprints/footprint_form_container/components/common/latest_projects_list/components/ProjectSummary.jsx

```diff
@@ -1,44 +1,70 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { useTexts } from '../../../../../contexts/TextsContext.js';
 
 const ProjectSummary = ({ project }) => {
   const [collapseState, setCollapseState] = useState('hidden');
+  const [shortText, setShortText] = useState(false);
```

Comment:
Jag tänkte först att denna innehöll en text, tills jag såg att det stod false, så för min del tror jag att det hade varit intuitivt att lägga till ordet "is" i början. Vad tror du?

```suggestion
  const [isShortText, setIsShortText] = useState(false);
```

Reply:
Like it